### ISSUE TITLE
Token Wizard no longer working in latest PLEX?

### ISSUE THREAD

**Issue by hepcat72** [HealsCodes/EyeContact#4]
I just installed PLEX for the first time and am trying to set up access to my EyeTV Hybrid.

First, I'm not sure if this is a problem, but I have PLEX set up only for local access (no remote access), so maybe that's my problem?  But...

I cannot find the "Channels" referred to in this portion of the config instructions:

> 1b) Configure your Plex EyeContact Plugin
> 
> open the Plugin inside the 'Video Channels' section of your PLEX client (eg. go to http://plex.tv/web, 'Channels', 'Eye Contact', 'Preferences' > Settings (gear icon))

The EyeContact plugin appears in Settings / Manage / Plugins in my locally accessed PLEX server (127.0.0.1...) and there's a spot for a token, but since I can't find the above, I have no token to enter.

Plus, even though I clear out the pin, when I go back, there's a pin there again - so could that be an issue?

**Comment by hepcat72** [comment]
Or... Is a PLEX pass required in order to have the "channels" thing mentioned?

**Comment by HealsCodes** [comment]
Honestly I'm surprised this is still compatible with PLEX at all I last touched it 8 years ago ^^,

**Comment by hepcat72** [comment]
Yeah, I figured it was a long shot, but I haven't found anything else to do it... (so far - just started looking into a solution this morning).

**Comment by HealsCodes** [comment]
I haven't used PLEX in a long timelbut if you install a plugin like youtube, where does it show up? I can imagine they simply moved or renamed "Channels" to something else.

**Comment by HealsCodes** [comment]
There being no alternative was my reason to write this plugin back then ;)

**Comment by hepcat72** [comment]
OK.  Reading between the lines, I think that I'm in the right place:

    Settings -> <Server Name> -> Manage -> Plugins

There, you can enter the settings mentioned in `1b) Configure your Plex EyeContact Plugin`, but after entering the settings and saving them, there is no apparent "token wizard" that I can find (as described) here:

> If you did it correctly, a new menu entry named 'Token Wizard' should appear above 'Settings' (or in newer versions of Plex, you will see a menu item in Plex called 'Preferences').

My instinct is that it should appear under:

    Settings -> <Server Name> -> Settings

I tried going here:

    Settings -> <Server Name> -> Settings -> Plugins

and checking `Disable capability checking` because the description seemed promising:

> Capability checking ensures that plug-ins that are incompatible with this version of the server or the current client application you are using are hidden. Disabling capability checking is useful during development, but will enable access to plug-ins that may perform unreliably with certain client applications.

but no dice...

**Comment by hepcat72** [comment]
I tried running the token gathering code on the command line, but after a minute or so waiting, it returns "No token":
```
python ~/Library/Application\ Support/Plex\ Media\ Server/Plug-ins/EyeContact.bundle/Contents/Libraries/Shared/tokenproxy/__init__.py
No Token
```
